controllers/manager/manager.go

```diff
@@ -645,36 +652,37 @@ func (m *PlatformManager) GetSystemInfo(namespace string, client *gophercloud.Se
 
 }
 
-func (m *PlatformManager) GetActiveHost(namespace string, client *gophercloud.ServiceClient) (*v1.Host, error) {
+func (m *PlatformManager) GetHostByPersonality(namespace string, client *gophercloud.ServiceClient, personality string) (*v1.Host, *hosts.Host, error) {
```

Comment:
The name of this method looks correct, prefer to have a unit test against it, nit: the return can be simplified as instance, host, err